src/Raven.Server/Rachis/Json/Sync/RachisBlittableJsonTextWriter.cs

```diff
@@ -14,6 +13,16 @@ public RachisBlittableJsonTextWriter(JsonOperationContext context, Stream stream
             _afterFlush = afterFlush;
         }
 
+        public void Dispose()
+        {
+            DisposeInternal();
+        }
+
+        public void Flush()
```

Comment:
Do we have any further usages of `AbstractBlittableJsonTextWriter` directly? 
If I call `AbstractBlittableJsonTextWriter.Flush` ?